src/plugins/tts/plugin.tts.js

```diff
@@ -233,12 +236,12 @@ BookReader.prototype.ttsStop = function () {
  * @param {PageChunk} chunk
  * @return {PromiseLike<void>} returns once the flip is done
  */
-BookReader.prototype.ttsBeforeChunkPlay = function(chunk) {
-  return this.ttsMaybeFlipToIndex(chunk.leafIndex)
-    .then(() => {
-      this.ttsHighlightChunk(chunk);
-      this.ttsScrollToChunk(chunk);
-    });
+BookReader.prototype.ttsBeforeChunkPlay = async function(chunk) {
+  await this.ttsMaybeFlipToIndex(chunk.leafIndex);
+  this.ttsHighlightChunk(chunk);
+  // This appears not to work; ttsMaybeFlipToIndex causes a scroll to the top of
+  // the active page :/ Disabling cause the extra scroll just adds an odd jitter.
+  // this.ttsScrollToChunk(chunk);
```

Comment:
do we want to keep this as memo? will it turn back on?

Reply:
I think so :/ We probably do want to re-enable this at some point. Otherwise as the TTS highlights the next chunk, it doesn't scroll to make sure it's in view :/